### PR TITLE
Landscape Modefor CompletedGame and CompletedScene for mobiles and tablets.

### PR DESCRIPTION
**Pull request by yatna**
Completed Scene:
screenshot_2016-01-28-03-17-36

Completed Game:
screenshot_2016-01-28-03-47-23

### CODE REVIEW

File: PowerUp/app/src/main/res/layout-land/completed_game.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    <TextView
+        android:id="@+id/completed_game_textView"
+        android:layout_width="@dimen/completed_game_width"
+        android:layout_height="@dimen/completed_game_height"
+        android:text="@string/completed_game"
+        android:textColor="@android:color/black"
+        android:textSize="@dimen/game_text_size"
+        android:layout_marginTop="@dimen/completed_game_textview"
+        android:layout_gravity="center_horizontal"
+        android:layout_weight="0.03" />
```
Comment (alma-castant) [inline PowerUp/app/src/main/res/layout-land/completed_game.xml]: Please don't put raw values directly in the layout file. Always use the dimensions file. Also consider whether you can do this by creating an alternate dimensions file only and not an alternate layout.

File: PowerUp/app/src/main/res/layout-land/completed_scene.xml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+    <TextView
+        android:id="@+id/completed_scene_textView"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:ems="10"
+        android:text="@string/completed_scene"
+        android:textColor="@android:color/black"
+        android:textSize="@dimen/game_text_size"
+        android:layout_above="@+id/ContinueButtonMap"
```
Comment (alma-castant) [inline PowerUp/app/src/main/res/layout-land/completed_scene.xml]: Only use `@+id` when declaring a new id. For references use `@id` only. Read the coding guidelines for the repo [here](https://github.com/systers/powerup-android#coding-guidelines)

File: PowerUp/app/src/main/res/layout-land/completed_scene.xml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+        android:layout_marginBottom="@dimen/completed_game_textview" />
+
+    <Button
+        android:id="@+id/ContinueButtonMap"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="@dimen/completed_game_textview"
+        android:text="@string/map"
+        android:layout_alignParentBottom="true"
+        android:layout_alignRight="@+id/completed_scene_textView"
```
Comment (alma-castant) [inline PowerUp/app/src/main/res/layout-land/completed_scene.xml]: Same

File: PowerUp/app/src/main/res/layout-land/completed_scene.xml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+        android:layout_alignEnd="@+id/completed_scene_textView" />
```
Comment (alma-castant) [inline PowerUp/app/src/main/res/layout-land/completed_scene.xml]: Same

**Comment by alma-castant** [comment]
I left comments on the code style. Also, please read the readme of the repository. You should only have one commit per PR, so please squash your two commits into one. Use a short, concise and meaningful name for your commit and write the rest of the details in the PR body so anyone can understand what it does. The branch has conflicts, so please resolve them after squashing your commits. You currently have a typo in your commit, so please fix it too ;)

**Comment by yatna** [comment]
Thanks for pointing out the mistakes. I have created a new PR keeping the necessay changes in mind.

**Pull request closed without merging**